engine/access/ping/engine.go

```diff
@@ -85,10 +102,12 @@ func (e *Engine) startPing() {
 	}
 }
 
-// send ping to a given node and report the reachable result to metrics
+// pingNode pings the given peer and updates the metrics with the result and the additional node information
 func (e *Engine) pingNode(peer *flow.Identity) {
-	reachable := e.pingAddress(peer.ID())
-	e.metrics.NodeReachable(peer, reachable)
+	id := peer.ID()
+	reachable := e.pingAddress(id)
+	info := e.nodeInfo[id]
```

Comment:
what if the id is not found in the node info map?

Reply:
then the `nodeinfo` reported is empty `""`.

Reply:
I thought of putting in something `unknown` but then settled for just letting it be empty.